File: app/src/main/java/by/nts/cafe/app/presentation/ui/halls/IHallsView.java
package by.nts.cafe.app.presentation.ui.halls;

import java.util.List;

import by.nts.cafe.app.model.db.HallModel;

public interface IHallsView {
    void onHallsLoaded(List<HallModel> hallsList);
    void onError(Throwable th);
    void showUpdateProcess(boolean isShow);
}

File: app/src/main/java/by/nts/cafe/app/presentation/ui/halls/IHallsView.kt
package by.nts.cafe.app.presentation.ui.halls

import by.nts.cafe.app.model.db.HallModel

interface IHallsView {
    fun onHallsLoaded(hallsList: List<HallModel>)
    fun onError(th: Throwable)
    fun showUpdateProcess(isShow: Boolean)
}
